opentelemetry-instrumentation/src/opentelemetry/instrumentation/bootstrap_gen.py

```diff
@@ -0,0 +1,138 @@
+# Copyright The OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
```

Comment:
Should this be here?

Reply:
Hm, sorry, why shouldn't it be here?

Reply:
I think it's because we removed the `opentelemetry-instrumentation` package in #595